src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,33 @@
+package racingcar;
+
+import java.util.Random;
+
+public class Car {
+	private static int MAXOFRANDOM = 10;
+	private static final int MOVINGCONDITION = 4;
+	private final String name;
+	private Position position;
```

Comment:
상수의 이름은 대문자와 언더바(언더스코어)를 활용하시면 좋을 것 같아요!
클래스변수와 인스턴스변수를 개행으로 구분하면 가독성에 도움이 될 것 같아요 😃 
Position 클래스를 분리한 것처럼 name도 분리하면 어떨까요?
```suggestion
	private static final int MAX_OF_RANDOM = 10;
	private static final int MOVING_CONDITION = 4;
	
	private final CarName name;
	private Position position;
```